ui/lib/sync/addon/components/secrets/destination-header.hbs

```diff
@@ -13,6 +13,35 @@
   }}
 />
 
+{{#if @destination.purgeInitiatedAt}}
```

Comment:
According to backend folks, there will always be a timestamp whether or not `purgeError` is returned, so only checking for this param on the model